lib-find-meal/src/main/java/com/kinandcarta/lib/find/meal/viewmodel/FindMealViewModel.kt

```diff
@@ -26,10 +27,12 @@ class FindMealViewModel @ViewModelInject constructor(
         object LoadingMeals : State()
         data class LoadedMeals(val meals: Meals, val distanceUnit: DistanceUnit) : State()
         data class Failed(val failure: Failure) : State()
+        data class ReservedMeal(val meal: Meal) : State()
```

Comment:
Very very minor but i tend to keep failure as the last case/state, and then keep the other states as close to the order they'll be encountered (i.e loading, loaded, reserved, failed)